apps/frontend/src/components/filters.tsx

```diff
@@ -49,10 +49,14 @@ const Filters = ({ filters, onFilterChange }: FiltersProps) => {
     const newMinPrice = +event.target.value;
     if (event.target.value === "") {
       setPriceError("");
+      setMinPrice(null);
     } else if (!regex.test(event.target.value)) {
       setPriceError(
         "This price is not valid, please make sure the value is positive and in the form xx.xx",
       );
+    } else if (newMinPrice === null) {
+      setPriceError("");
+      setMinPrice(null);
```

Comment:
What is the case where newMinPrice is null and not ""?

Reply:
Since the number is accepted in a text box when nothing is entered the `event.target.value` is "" but the minPrice can not hold that value so when a search happens and I am setting the value to null to show the empty string. This is why I am checking both.